build.sbt

```diff
@@ -9,7 +9,7 @@ import com.typesafe.tools.mima.plugin.MimaKeys.{
 ThisBuild / scalaVersion := "2.12.12"
 
 ThisBuild / crossScalaVersions := Seq(
-  "2.11.12", (scalaVersion in ThisBuild).value, "2.13.3")
+  "2.11.12", (scalaVersion in ThisBuild).value, "2.13.4")
```

Comment:
not sure if this PR is the right time to upgrade this

I've always the reflex of upgrading patch semver when I can, let me now if it should not be done here